bin/node/rpc-client/Cargo.toml

```
@@ -12,7 +12,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 futures = "0.1.29"
-hyper = "0.12.35"
+hyper = "0.13.9"
```

```suggestion
hyper = "~0.12.35"
```

Maybe if we do this, dependabot will not try to update the version?